spec/ddtrace/contrib/concurrent_ruby/integration_test_spec.rb

```diff
@@ -4,57 +4,52 @@
 require 'ddtrace'
 
 RSpec.describe 'ConcurrentRuby integration tests' do
-  around do |example|
-    unmodified_future = ::Concurrent::Future.dup
-    example.run
+  # DEV We save an unmodified copy of Concurrent::Future.
+  let!(:unmodified_future) { ::Concurrent::Future.dup }
+
+  # DEV We then restore Concurrent::Future, a dangerous game.
+  after do
     ::Concurrent.send(:remove_const, :Future)
     ::Concurrent.const_set('Future', unmodified_future)
     remove_patch!(:concurrent_ruby)
   end
 
-  let(:tracer) { get_test_tracer }
-  let(:configuration_options) { { tracer: tracer } }
+  let(:configuration_options) { {} }
 
-  subject(:deferred_execution) do
+  subject do
```

Comment:
Why an anonymous subject? Unless there's a functional reason why we can't name it, we should generally give it a name.